.gitignore

```diff
@@ -2,8 +2,10 @@
 .idea/
 .nyc_output/
 .test_results/
+.npmignore
```

Comment:
why would you put .npmignore in .gitignore?

Reply:
Removed